app/views/sprint-6/book-and-manage/make-a-referral/referral-task-list.html

```diff
@@ -178,169 +178,173 @@ <h2 class="moj-task-list__section">
-              {% else %}
-                <span class="app-task-list__task-name">
-                  Select required complexity level
-                </span>
-              {% endif %}
-            </li>
-            <li class="moj-task-list__item">
-              {% if data["accommodation-desired-outcomes"] %}
-                <a class="moj-task-list__task-name" href="accommodation-service/completion-date">What date does the accommodation service need to be completed by?</a>
-              {% else %}
-                <span class="app-task-list__task-name">
-                  What date does the accommodation service need to be completed by?
-                </span>
-              {% endif %}
-            </li>
-            <li class="moj-task-list__item">
-              {% if data["accommodation-date-year"] %}
-                <a class="moj-task-list__task-name" href="accommodation-service/rar-days">Enter RAR days used</a>
-              {% else %}
-                <span class="app-task-list__task-name">
-                  Enter RAR days used
-                </span>
-              {% endif %}
-            </li>
-            <li class="moj-task-list__item">
-              {% if data["accommodation-using-rar-days"] %}
-                <a class="moj-task-list__task-name" href="accommodation-service/further-information">Further information for service provider</a>
-              {% else %}
-                <span class="app-task-list__task-name">
-                  Further information for service provider
-                </span>
-              {% endif %}
-            </li>
-          </ul>
-        </li>
+        {% if data["found-interventions"] %}
+          <li>
+            {% if data["accommodation-more-info-completed"] %}
+              <strong class="govuk-tag govuk-tag app-task-list__tag">
+                completed
+              </strong>
+            {% elseif data["accommodation-sentence"] %}
+              <strong class="govuk-tag govuk-tag app-task-list__tag">
+                in progress
+              </strong>
+            {% elseif data["needs-and-requirements-completed"] %}
+              <strong class="govuk-tag govuk-tag--grey app-task-list__tag">
+                not started
+              </strong>
+            {% else %}
+              <strong class="govuk-tag govuk-tag--grey app-task-list__tag">
+                cannot start yet
+              </strong>
+            {% endif %}
+            <h2 class="moj-task-list__section">
+              <span class="moj-task-list__section-number">5.
+              </span>
+              Accommodation referral
+            </h2>
+            <ul class="moj-task-list__items">
+              <li class="moj-task-list__item">
+                {% if data["needs-and-requirements-completed"] %}
+                  <a class="moj-task-list__task-name" href="accommodation-service/select-sentence">Select relevant sentence</a>
```

Comment:
For the social inclusion sentence, we say "Select relevant sentence for the social inclusion service referral", do we want to be consistent here (and say "for the accommodation service referral")?

Reply:
Good shout, changed it.